lessons/Data-Science/lookups/langs/en-us/pages/lookup-expressions.adoc

```diff
@@ -0,0 +1,60 @@
+= Lookup Expressions
+++++
+<style>
+/* For the example prompt, keep the lines tight and centered */
+.tightAndCentered * { padding: 0 !important; text-align: center; }
+</style>
+++++
+
+Make sure you've opened your *saved copy* of the @starter-file{animals}, in which you've defined the sample Rows from @printable-exercise{defining-rows.adoc}, such as `lizard-row`, `old-row`, etc.
+
+[.tightAndCentered]
+The code below draws a solid, green triangle, using 5x the age of `old-row` as the size: +
```

Comment:
The description and the code don't match here. Please decide whether you want it to use the age or 5x the age.